pytrip/cube.py

```diff
@@ -495,11 +500,11 @@ def _read_trip_header_file(self, header_path):  # TODO: could be made private? #
         # load plain of gzipped file
         if header_path.endswith(".gz"):
             import gzip
-            fp = gzip.open(header_path, "rt")
+            with gzip.open(header_path, "rt") as fp:
```

Comment:
Will it work for python 2 ?

Reply:
Changed it to reading binary and then decoding the data, so that it works with both 2 and 3.